src/cogs/PrequisiteChecker.py

```diff
@@ -13,8 +15,40 @@ def __init__(self, client):
         self.client = client
 
     @commands.command()
-    async def ping(self, ctx):
-        await ctx.send("Pong!")
+    async def prereq(self, ctx, arg):
+
+        # A lot of error checking and log messages
+        if len(arg) != 7:
+            await ctx.send("Invalid Input")
+            return
+
+        program = arg[0:4]
+        if program != "cpen" and program != "elec":
+            await ctx.send("Unable to identify specified program")
+            return
+
+        courseNumString = arg[4:7]
```

Comment:
Likewise, this needs to account for courses like 400M, 400D, and 400A.

Reply:
Forgot to mention, but can you also change the variable names to use `snake_case` here?